packages/cli/lib/publicApps.js

```diff
@@ -0,0 +1,60 @@
+const {
+  fetchPublicAppOptions,
+  selectPublicAppPrompt,
+} = require('./prompts/selectPublicAppPrompt');
+const { EXIT_CODES } = require('./enums/exitCodes');
+const { i18n } = require('./lang');
+const { uiLine } = require('./ui');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const {
+  migrateApp,
+  checkMigrationStatus,
+} = require('@hubspot/local-dev-lib/api/projects');
+
+const i18nKey = 'cli.lib.publicApps';
+
+const fetchPublicApp = async (accountId, accountName, options, migrateApp) => {
+  const { appId } = await selectPublicAppPrompt({
+    accountId,
+    accountName,
+    options,
+    migrateApp,
+  });
+  return appId;
+};
+
+const migratePublicApp = async (accountId, appId, name) => {
```

Comment:
Seems like most of the utils in here are very thin wrappers around api calls from local dev lib. IMO most of the utils in here aren't necessary as they're adding a layer of abstraction without providing much value. The one that seems most useful is `validateAppId`, but since it's only used it one place I think it would be fine to live in migrateApp.js

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/5652758620136accfe8bedcb6f880ea864605757